tests/numl.test.js

```diff
@@ -374,4 +380,63 @@ describe("numl", function() {
     .should.throw(
       `SyntaxError: Unrecognised type "bad-type".`);
   });
+
+  it("should parse text properties", function() {
+    numl(`
+      # _
+
+      ## _
+
+      ### _
+      foo: \`bar baz\`
+      baz: \`
+        quux
+          corge
+            grault
+      \`
+      garply[text]: \`
+        waldo
+          fred
+      \`
+      rar [text]: \`
+        23-!@$%^&*_ rar ポケモン
+      \`
+      lorem: \`\`
+    `)
+    .should.shallowDeepEqual({
+      sequences: [{
+        blocks: [{
+          properties: {
+            foo: {
+              __type__: 'text',
+              value: 'bar baz',
+            },
+            baz: {
+              __type__: 'text',
+              value: str`
```

Comment:
I don't even want to know what Javascript language writers thought this feature was necessary. :)